### DISCUSSION TITLE
Increase resolution of a previous image.

### DISCUSSION THREAD

**Discussion by dgrogandesign** [carson-katri/dream-textures#142] [Q&A]
Hi there, thanks for creating such an incredible piece of software! Is there any way to recreate the same image but again but with a different resolution? I ask because it is faster to generate lower resolution images but would like to generate higher resolution ones once i have found one I like. Many thanks, Danny

**Comment by NullSenseStudio** [comment]
Unfortunately due to how stable diffusion works changing anything (including the resolution) will change the image. There are plans to support AI upscaling eventually.
